OpenRA.Mods.Common/Traits/World/OrderEffects.cs

```diff
@@ -50,6 +53,9 @@ public class OrderEffectsInfo : TraitInfo
 		[Desc("Number of ticks between (frozen) actor flashes.")]
 		public readonly int ActorFlashInterval = 2;
 
+		[Desc("Order name(s) this applies for (empty if applies to all).")]
+		public readonly List<string> OrderScope = new();
```

Comment:
Maybe `AllowedOrders` or `ValidOrders`? The `Allowed` and `Valid` prefixes are used in other places of the engine.